backend/src/libs/packages/encrypt/encrypt.package.ts

```diff
@@ -0,0 +1,27 @@
+import bcrypt from 'bcryptjs';
+
+import { type IEncryptService } from '~/libs/interfaces/encrypt.interface';
+import { type HashedPass } from '~/packages/auth/libs/types/hashed-pass.type';
```

Comment:
Let's add extensions in one of the next tasks.
```suggestion
import { type IEncryptService } from '~/libs/interfaces/encrypt.interface.js';
import { type HashedPass } from '~/packages/auth/libs/types/hashed-pass.type.js';
```

Note to myself: should be handled by linter, why it's not working?